lib/srv/uacc/uacc_linux.go

```diff
@@ -0,0 +1,128 @@
+// +build linux
+
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package uacc
+
+// #include <stdlib.h>
+// #include "uacc.h"
+import "C"
+
+import (
+	"encoding/binary"
+	"net"
+	"sync"
+	"unsafe"
+
+	"github.com/gravitational/trace"
+)
+
+// This module concerns itself with updating the user account database and log on nodes
+// that a client connects to with an interactive session.
```

Comment:
make this a package doc comment https://golang.org/doc/effective_go.html#commentary

also, in Go "module" has a slightly different meaning.
this is a "package", and "module" is a versioned collection of packages ;)